libs/components/forms/src/lib/modules/checkbox/checkbox.component.ts

```diff
@@ -46,48 +47,71 @@ export class SkyCheckboxComponent implements ControlValueAccessor, OnInit {
    * If the checkbox does not include a visible label, use `label` instead.
    */
   @Input()
-  public labelledBy: string;
+  public labelledBy: string | undefined;
 
   /**
    * Specifies an ID for the checkbox.
    * @default a unique, auto-incrementing integer. For example: `sky-checkbox-1`
    */
+  @HostBinding('attr.id')
```

Comment:
What's the reasoning behind adding this value to the `sky-checkbox` element?

Reply:
Talked offline - removed